digits/dataset/images/classification/test_views.py

```diff
@@ -115,6 +115,10 @@ def create_dataset(cls, **kwargs):
         if request_json:
             url += '.json'
 
+        clone = data.pop('clone', None)
+        if clone is not None:
+            url += '?clone=' + clone
```

Comment:
I thought we said this was going to work for POST data as well as query string arguments? Why is this necessary?